src/Package/Test/DataInspect/EvaluationWrapperTest.cs

```diff
@@ -453,6 +453,41 @@ public class EvaluationWrapperTest {
 
         [Test]
         [Category.Variable.Explorer]
+        public async Task DataFrameNATest() {
```

Comment:
Can you make it a test in `GridDataTest` class instead? This one I consider legacy code, because it takes way more dependencies than it actually needs to tests what it's testing - the only reason why it's still around is because I didn't have time to port all the tests from here to there.